cmd/crossplane/core/core.go

```diff
@@ -105,7 +105,9 @@ type startCommand struct {
 	MaxReconcileRate                 int           `default:"100" help:"The global maximum rate per second at which resources may checked for drift from the desired state."`
 	MaxConcurrentPackageEstablishers int           `default:"10"  help:"The the maximum number of goroutines to use for establishing Providers, Configurations and Functions."`
 
-	WebhookEnabled bool `default:"true" env:"WEBHOOK_ENABLED" help:"Enable webhook configuration."`
+	WebhookEnabled                      bool `default:"true"  env:"WEBHOOK_ENABLED"                        hidden:""`
+	EnableWebhooks                      bool `default:"true"  env:"ENABLE_WEBHOOKS"                        help:"Enable webhook configuration."`
+	AutomaticDependencyDowngradeEnabled bool `default:"false" env:"AUTOMATIC_DEPENDENCY_DOWNGRADE_ENABLED" help:"Enable automatic dependency version downgrades. This configuration requires the 'EnableDependencyVersionUpgrades' feature flag to be enabled."`
```

Comment:
i'm not sure the rebase was done correctly, it looks like this `AutomaticDependencyDowngradeEnabled` flag is being re-added after @negz removed it in https://github.com/crossplane/crossplane/commit/0013a3fea4c605ae6e92ceb02640230320fb10e5#diff-20d88315192c28a14e8d19b9042c1dfbfad06ce54916ab4b387dda1aae5e2894L114.

This should not be added back, right?

Reply:
Removed it again.